src/main/java/com/networknt/schema/utils/DefaultNode.java

```diff
@@ -0,0 +1,19 @@
+package com.networknt.schema.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaRef;
+
+public class DefaultNode {
```

Comment:
Rename the class to Defaults and since you pulled this out add the appropriate unit test.